react-intro updates (#234)
patternfly/training-scenarios: react/react-basics/intro.md

> PatternFly React is made up of components, layouts and demos.

Reviewer: ```suggestion
PatternFly React is made up of components, layouts, and demos.
```

> The PatternFly React library provides a collection of React components used to build interfaces with consistent markup, styling, and behavior.  In this course, a PatternFly card is built starting with content, adding a header and action buttons, and concluding by adding a card footer.

Reviewer: ```suggestion
The PatternFly React library provides a collection of React components used to build interfaces with consistent markup, styling, and behavior.  In this course, you will build a PatternFly card starting with content, adding a header and action buttons, and concluding by adding a card footer.
``` 
Note:
For things that will happen in the future (like intros), we can use "will" for future tense (George and I touched base on this earlier via Slack).
Also, I left a comment about this in another area, but we don't need to avoid "you" everywhere. We should cut the pronoun when we can in instructions, like changing "You can click the table" to "Click the table." But it's okay to use "you" elsewhere when we need to.
